.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "bibdk_development"]
+	path = bibdk_development
```

Comment:
skal path ikke bare være 'development'? evt. 'development_setup'?
Såvidt jeg kan gennemskue, er ideen at vi samler development setup for alle applikationer i https://gitlab.dbc.dk/d-scrum/docker/development_setup repo'et? 
Så vil path blive development/bibdk

Reply:
Det lyder meget fornuftigt.